src/celeritas/ext/detail/GeantPhysicsList.cc

```diff
@@ -286,6 +291,13 @@ void GeantPhysicsList::add_e_processes(G4ParticleDefinition* p)
 
         CELER_LOG(debug) << "Loaded Coulomb scattering with "
                             "G4eCoulombScatteringModel";
+        if (options_.msc == MscModelSelection::wentzel_vi
+            || options_.msc == MscModelSelection::urban_wentzel)
+        {
+            CELER_LOG(warning)
+                << "Coulomb scattering may be inconsistent with msc="
+                << to_cstring(options_.msc);
```

Comment:
I think it's ok to use Coulomb scattering in combination with MSC.

Reply:
I thought that high-energy MSC was decomposed into a discrete process (high-angle coulomb scattering) and the continuous MSC process (Wenzel-VI). AdePT replaces both of them with Urban. Wouldn't it then be inconsistent to have coulomb in addition to Urban MSC?

Reply:
But aren't you warning if we have Coulomb + Wenzel-VI, not Coulomb + Urban here?

Reply:
Yep, I'm dumb. Meant this to be urban + extended.